Jellyfin.Server/Program.cs

```diff
@@ -224,7 +224,7 @@ private static async Task StartApp(StartupOptions options)
             {
                 _logger.LogInformation("Running query planner optimizations in the database... This might take a while");
                 // Run before disposing the application
-                using var context = new JellyfinDbProvider(appHost.ServiceProvider, appPaths).CreateContext();
+                using var context = new JellyfinDbProvider(appHost.ServiceProvider, appPaths, (ILogger<JellyfinDbProvider>)_logger).CreateContext();
```

Comment:
JellyfinDbProvider is already registered in ServiceProvider, so you could just call `appHost.Resolve<JellyfinDbProvider>()`